src/test/java/com/mongodb/jdbc/MongoSQLMock.java

```diff
@@ -35,10 +35,12 @@
     protected static int INT_COL = 6;
     // foo.d
     protected static int ANY_COL = 7;
+    // foo.doc
+    protected static int DOC_COL = 8;
```

Comment:
the labels have to be alphabetically ordered, which is why this is the new 8 as opposed to being introduced as 10